internal/evaluation_target/application_snapshot_image/application_snapshot_image.go

```diff
@@ -352,27 +346,8 @@ type attestationExtraData struct {
 // a standard process for Marshaling the JSON can be used, thus removing the need for this method.
 func (a attestationData) MarshalJSON() ([]byte, error) {
 	buffy := bytes.Buffer{}
-	raw, err := a.RawMessage.MarshalJSON()
-	if err != nil {
-		return nil, err
-	}
-
-	if _, err = buffy.Write(raw[0 : len(raw)-1]); err != nil {
-		return nil, err
-	}
-
-	if _, err = buffy.WriteString(`,"extra":`); err != nil {
-		return nil, fmt.Errorf("write extra key: %w", err)
-	}
-	extra, err := json.Marshal(a.Extra)
-	if err != nil {
-		return nil, fmt.Errorf("marshal json extra: %w", err)
-	}
-	if _, err := buffy.Write(extra); err != nil {
-		return nil, fmt.Errorf("write extra value: %w", err)
-	}
 
-	_, err = buffy.WriteString(`, "statement":`)
+	_, err := buffy.WriteString(`, "statement":`)
```

Comment:
This is odd. Should this be `{"statement":` since this is now the first thing written?

Reply:
Ah! The unit tests are catching this as an error

Reply:
I knew I was missing something. Fixed!

Reply:
Github didn't refresh and I only caught your first comment before I pushed the change :sweat_smile:

Reply:
It's still writing a leading comma